executors/stable/executor.py

```diff
@@ -45,6 +46,7 @@ class StableDiffusionGenerator(Executor):
 
     def __init__(self,
         batch_size: int=4,
+        config_path: Optional[str]=None,
```

Comment:
lets move this at the end of the argument so that it is not a breaking change

Reply:
OK